setup.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from setuptools import find_namespace_packages, setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name="amazon-braket-examples",
+    license="Apache License 2.0",
+    python_requires=">= 3.7",
+    packages=find_namespace_packages(where="examples", exclude=("tests",)),
+    package_dir={"": "examples"},
+    install_requires=[
+        "ipykernel",
+        "boto3",
+        "amazon-braket-sdk",
+        "amazon-braket-default-simulator",
+        "dwave-ocean-sdk",
+        "amazon-braket-ocean-plugin",
+        "amazon-braket-pennylane-plugin",
+        "amazon-braket-schemas",
+        "PennyLane",
+    ],
+    extras_require={
+        "test": [
+            "black",
+            "flake8",
+            "nbconvert",
+            "isort",
+            "pre-commit",
+            "pylint",
+            "pytest",
+            "pytest-cov",
+            "pytest-rerunfailures",
+            "pytest-xdist",
+            "tox",
+        ]
+    },
+    url="https://github.com/aws/amazon-braket-examples",
+    author="Amazon Web Services",
+    description=(
+        "An open source library for tutorials on quantum computing using Amazon Braket."
+    ),
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    keywords="Amazon AWS Quantum",
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Natural Language :: English",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+    ],
```

Comment:
I don't think these are necessary, since nothing is being installed. In fact, is setup.py needed at all?

Reply:
tox.ini is not initiating without this

Reply:
Ah, in that case just delete these lines; they're only useful for distribution ie pypi